tests/xharness/Jenkins/MacTestTasksFactory.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using Microsoft.DotNet.XHarness.iOS.Shared;
+using Microsoft.DotNet.XHarness.iOS.Shared.Execution;
+using Microsoft.DotNet.XHarness.iOS.Shared.Utilities;
+using Xharness.Jenkins.TestTasks;
+
+namespace Xharness.Jenkins {
+	class MacTestTasksFactory : IEnumerable<RunTestTask> {
+
+		readonly Jenkins jenkins;
+		readonly IProcessManager processManager;
+		readonly ICrashSnapshotReporterFactory crashReportSnapshotFactory;
+		readonly ITestVariationsFactory testVariationsFactory;
+
+		public MacTestTasksFactory (Jenkins jenkins, 
+								    IProcessManager processManager,
```

Comment:
Indentation looks funny.

Reply:
I think is GitHub getting confused, look ok in my editor, will double check.